### ISSUE TITLE
End of life program for the integration of Skype for Business with third-party audio conferencing providers

### ISSUE THREAD

**Issue by hmank** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#115]
The end-of-integration-with-3rd-party-providers should be in a different section as it does not fall into the Accessibility-and-Regulatory category, it is more suited under a Life-cycle subsection to Skype for Business ie.  https://docs.microsoft.com/en-us/SkypeForBusiness/Life-cycle/end-of-integration-with-3rd-party-providers

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7d025b7c-a120-e129-ea13-af2c0369dcd0
* Version Independent ID: 2c2118a8-277a-d56b-18a5-b6c4ccd3fb8b
* Content: [End of life program for the integration of Skype for Business with third-party audio conferencing providers ](https://docs.microsoft.com/en-us/SkypeForBusiness/accessibility-and-regulatory/end-of-integration-with-3rd-party-providers)
* Content Source: [Skype/SfbOnline/accessibility-and-regulatory/end-of-integration-with-3rd-party-providers.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Skype/SfbOnline/accessibility-and-regulatory/end-of-integration-with-3rd-party-providers.md)
* Service: **skype-for-business-online**
* GitHub Login: @tonysmit
* Microsoft Alias: **tonysmit**

**Comment by tonysmit** [comment]
Sorry, that's the node I have for these types of things. I will think about renaming that node. Thanks for the feedback.
